.changeset/seven-words-trade.md

```diff
@@ -0,0 +1,13 @@
+---
+"@biomejs/biome": minor
```

Comment:
I don't think this needs to be a minor. since it's new syntax that will now be parsed successfully, it will ideally reduce errors encountered by users.

```suggestion
"@biomejs/biome": patch
```